README.md

```diff
@@ -147,7 +147,7 @@ or
 * * Go to lambda, click on **Test**, and paste this json:
 ```json
 {
-  "queryStringParameters": {"path": __YOUR_IMAGE_PATH_WITH_SIZE_PREFIX__}
+  "queryStringParameters": {"path": "__YOUR_IMAGE_PATH_WITH_SIZE_PREFIX__"}
```

Comment:
I intentionally have written the value that way.
Firstly, it's red and captures a look immediately.
Secondly, even if user copies and pastes it as it is, it is invalid **json**, so it going to throw an error.

However, in case you are able to give a good explanation why the changed code is better, we can continue.

Reply:
ah I see - it caused me an invalid JSON as well that's why I wanted to fix it. But your reasoning makes sense! I'll revert that change :)